src/utils/__tests__/__snapshots__/createSchema.test.ts.snap

```diff
@@ -15,6 +15,12 @@ type Collection {
   # Description of the collection which can include links to other collections
   description: String
 
+  # Markdown alternate of description field contents.
+  descriptionMarkdown: String
+
+  # Collection has its description available as markdown
+  hasMarkdownDescription: Boolean!
```

Comment:
What's the use case for this field?

If we're going with the approach in this PR, wouldn't we have run a migration beforehand so that `hasMarkdownDescription` would always be `true`? (Or at least true when there is any description at all?)

Reply:
good point!

Reply:
@ashleyjelks what do you think? Should we remove this boolean, or are you using it reaction? We'd need another PR to MP to pull it out which should not be a factor but I'm eyeing it sideways anyway...

Reply:
I think we should keep it bc in Reaction we need to handle the in-betweens states that some collections will be in where it needs to render HTML bc the data re-processing hasn't isn't complete, and also bc some collections simply don't have descriptions so we can default to render nothing when the boolean is false.

Reply:
I'd be surprised if that text processing took more than a matter of minutes, if not seconds. For that reason this field strikes me as overkill. 

Also I'd say we should be able gracefully handle the absence of any field on the client side, without needing an _extra_ field to tell us whether or not that field is present? Is there something special about the description field in Emission that would require us to have that "default to render nothing" behavior?

Reply:
> we should be able gracefully handle the absence of any field on the client side

Agreed!

Since Myk is ooo the rest of the day and I'd like to try to move this work forward, does anyone object to removing the boolean field in a follow-up PR? We think this is safe to merge? cc: @mzikherman @anandaroop